test/modules/linear_boltzmann_solvers/transport_steady/reed_balance.lua

```diff
@@ -0,0 +1,100 @@
+-- Standard Reed 1D 1-group problem
+
+-- Create Mesh
+widths = {2., 1., 2., 1., 2.}
+nrefs  = {200,200,200,200,200}
+
+Nmat = #widths
+
+nodes = {}
+counter = 1
+nodes[counter] = 0.
+for imat = 1, Nmat do
+    dx = widths[imat]/nrefs[imat]
+    for i = 1, nrefs[imat] do
+        counter = counter + 1
+        nodes[counter] = nodes[counter-1] + dx
+    end
+end
+
+meshgen = mesh.OrthogonalMeshGenerator.Create({ node_sets = {nodes} })
+mesh.MeshGenerator.Execute(meshgen)
+
+-- Set Material IDs
+z_min = 0.0
+z_max = widths[1]
+for imat = 1, Nmat do
+    z_max = z_min + widths[imat]
+    log.Log(LOG_0, 'imat='..imat..', zmin='..z_min..', zmax='..z_max)
+    lv = logvol.RPPLogicalVolume.Create(
```

Comment:
Inconsistent indentation. I like to do this:
```
lv = logvol.RPPLogicalVolume.Create({
    infx=true,
    infy=true,
    zmin=z_min, 
    zmax=z_max
})
```

Reply:
I used Daryl's lua-format config file and used the LuaFormatter he suggested